libs/ec/inc/public/tfc/devices/base.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <boost/asio.hpp>
```

Comment:
I think you can include `boost/asio/io_context.hpp` which should reduce the include size